units/host_drawdown.go

```diff
@@ -87,7 +89,16 @@ func (j *hostDrawdownJob) Run(ctx context.Context) {
 		if drawdownTarget <= 0 {
 			break
 		}
-		j.AddError(j.checkAndTerminateHost(ctx, &idleHost, &drawdownTarget))
+		err = j.checkAndTerminateHost(ctx, &idleHost, &drawdownTarget)
+		if err != nil {
+			grip.Error(message.Fields{
```

Comment:
Instead of putting error in the `message.Fields`, can write this as:
````go
grip.Error(message.WrapError(err, message.Fields {
<other metadata fields>
}))
```
This will standardize it against the other logging message formats and includes the stack trace, which helps sometimes.